book/animations.md

```diff
@@ -2283,8 +2292,8 @@ CSS property and parsing of `@keyframe` to implement the demos
 [here](examples/example13-opacity-animation.html) and
 [here](examples/example13-width-animation.html).
 
-13-6 *Overlap testing w/transform animations*. (If you've already done the
-transform animations exercise.) Our
+13-6 *Overlap testing w/transform animations*. (If you've already done
```

Comment:
I think this is the only use of `w/` in the book. Can we rename somehow?

Reply:
Also starting with a parenthetical is also unique. Can we move the parenthetical one sentence later and rephrase as something like "You'll need to have done Exercise 13-3"?